src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -88,11 +90,42 @@ object Run  {
   trait TerminalRunStatus extends RunStatus
   case object Initializing extends RunStatus
   case object Running extends RunStatus
-  case object Success extends TerminalRunStatus
+  case class Success(events: Seq[ExecutionEventEntry]) extends TerminalRunStatus
   final case class Failed(errorCode: Int, errorMessage: String) extends TerminalRunStatus {
     // Don't want to include errorMessage or code in the snappy status toString:
     override def toString = "Failed"
   }
+
+  // An event with a startTime timestamp
+  private case class EventStartTime(name: String, timestamp: DateTime)
+
+  def getEventList(op: Operation): Seq[ExecutionEventEntry] = {
+
+    val starterEvents = eventIfExists("createTime", op, "chilling out, maxing") ++ eventIfExists("startTime", op, "relaxing all cool")
+
+    val eventsList: Seq[EventStartTime] = if (op.getMetadata.containsKey("events")) {
+      op.getMetadata.get("events").asInstanceOf[java.util.ArrayList[AnyRef]].asScala map { x =>
```

Comment:
I don't disbelieve it's necessary but can you explain why the casting is necessary?